source/magpie/dataframe.d

```diff
@@ -767,6 +767,90 @@ public:
         indx.optimize();
     }
 
+    /++
+    from_csv rebuild for faster read
+    +/
+    void fastCSV(string path, size_t indexDepth, size_t columnDepth, char sep = ',')
+    {
+        import std.array: appender, split;
+        import std.stdio: File;
+        import std.string: chomp;
+
+        File csvfile = File(path, "r");
+        string[][] lines;
+        int totalLines = 0;
+
+        while(!csvfile.eof())
+        {
+            ++lines.length;
+            lines[totalLines++] = chomp(csvfile.readln()).split(sep);
```

Comment:
you could write this as `csvfile.readln.chomp.split(sep)`, but `File` provides a `byLine` range which would allow you to write `csvfile.byLine.chomp.split(sep).array;` in place of the loop.

Reply:
Done in dec86d1
[Thanks for the help :sweat_smile: ]